components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1673,15 +1688,23 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
                                 >
                                     <IconContainer
                                         ref={this.postPriorityPickerRef}
+                                        style={{position: 'relative'}}
```

Comment:
This should probably be applied via CSS instead of using the style tag. Could we maybe just apply it to all  `IconContainer` instances?